roles/default/kiali-deploy/tasks/main.yml

```
@@ -333,19 +333,26 @@
   - kiali_vars.auth.strategy != 'token'
 - name: Confirm auth strategy is valid for Kubernetes environments
   fail:
-    msg: "Invalid auth.strategy [{{ kiali_vars.auth.strategy }}]! Must be one of either 'login', 'ldap', 'token' or 'anonymous'"
+    msg: "Invalid auth.strategy [{{ kiali_vars.auth.strategy }}]! Must be one of either 'login', 'ldap', 'token', 'openid' or 'anonymous'"
   when:
   - is_k8s == True
   - kiali_vars.auth.strategy != 'login'
   - kiali_vars.auth.strategy != 'anonymous'
   - kiali_vars.auth.strategy != 'ldap'
   - kiali_vars.auth.strategy != 'token'
+  - kiali_vars.auth.strategy != 'openid'
 - name: Confirm ldap configuration when auth strategy is 'ldap'
   fail:
     msg: "Invalid configuration for LDAP! The mandatory parameters should be provided like `ldap_host', 'ldap_port', 'ldap_bind_dn', and 'ldap_base'"
```

I know you didn't change this.. but.. see below.
```suggestion
    msg: "Invalid configuration for LDAP! The mandatory parameters should be provided like 'ldap_host', 'ldap_port', 'ldap_bind_dn', and 'ldap_base'"
```

---

And fixed this backtick too